src/XMakeBuildEngine/UnitTests/BackEnd/LoggingService_Tests.cs

```diff
@@ -381,7 +381,7 @@ public void RegisterDistributedLoggerServiceShutdown()
 #if FEATURE_ASSEMBLY_LOCATION
                 LoggerDescription description = CreateLoggerDescription(className, Assembly.GetAssembly(typeof(ProjectCollection)).FullName, true);
 #else
-            LoggerDescription description = CreateLoggerDescription(className, typeof(ProjectCollection).GetTypeInfo().Assembly.FullName, true);
+                LoggerDescription description = CreateLoggerDescription(className, typeof(ProjectCollection).GetTypeInfo().Assembly.FullName, true);
```

Comment:
Low-pri but I'll ask anyway:  can you move the whitespace-only changes into their own commit so future history spelunking is easier?